[2022-12-11 20:00:20,120] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.create_task_table manual__2022-12-11T19:59:58.361670+00:00 [queued]>
[2022-12-11 20:00:20,155] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.create_task_table manual__2022-12-11T19:59:58.361670+00:00 [queued]>
[2022-12-11 20:00:20,156] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 20:00:20,157] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-11 20:00:20,157] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 20:00:20,199] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): create_task_table> on 2022-12-11 19:59:58.361670+00:00
[2022-12-11 20:00:20,211] {standard_task_runner.py:52} INFO - Started process 5420 to run task
[2022-12-11 20:00:20,231] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'create_task_table', 'manual__2022-12-11T19:59:58.361670+00:00', '--job-id', '1326', '--raw', '--subdir', 'DAGS_FOLDER/first-dag.py', '--cfg-path', '/tmp/tmp_u_xxrjc', '--error-file', '/tmp/tmp1p42kcyl']
[2022-12-11 20:00:20,234] {standard_task_runner.py:80} INFO - Job 1326: Subtask create_task_table
[2022-12-11 20:00:20,846] {task_command.py:369} INFO - Running <TaskInstance: first_dag.create_task_table manual__2022-12-11T19:59:58.361670+00:00 [running]> on host 7e39590a7807
[2022-12-11 20:00:23,451] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=create_task_table
AIRFLOW_CTX_EXECUTION_DATE=2022-12-11T19:59:58.361670+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-11T19:59:58.361670+00:00
[2022-12-11 20:00:23,722] {base.py:68} INFO - Using connection ID 'db' for task execution.
[2022-12-11 20:00:23,925] {dbapi.py:208} INFO - Running statement: 
                create table if not exists task (
	            value_1 INT,
	            value_2 INT
            );
              , parameters: None
[2022-12-11 20:00:24,355] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=create_task_table, execution_date=20221211T195958, start_date=20221211T200020, end_date=20221211T200024
[2022-12-11 20:00:24,399] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-11 20:00:24,597] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
